lib/dpla/map/factories.rb

```diff
@@ -10,7 +10,7 @@
     date { |timespan| timespan.association :timespan, :strategy => :build }
     description 'Window of the Stonewall Bar N.Y. 1969. The other half of the graffiti was erased by the time Diana photographed it.'
     extent '10x12 cm'
-    format 'Silver Gelatin Print'
+    dcformat 'Silver Gelatin Print'
```

Comment:
I see what you mean now.  A couple of options:

  - since we know that this gem needs to support `method_missing` on property names, we could add verification that properties don't cause conflicts.
  - we could use `#add_property` and `#add_child` directly in Krikri when conflicts occur. https://github.com/dpla/KriKri/blob/develop/lib/krikri/mapping_dsl.rb#L36